scripts/collect.ts

```diff
@@ -0,0 +1,323 @@
+import fs from "node:fs/promises";
+import { Readable } from "node:stream";
+import { pipeline } from "node:stream/promises";
+import { App, Octokit } from "octokit";
+import { config } from "dotenv";
+import { paginateRest } from "@octokit/plugin-paginate-rest";
+import { Semaphore } from "@core/asyncutil";
+import * as logtape from "@logtape/logtape";
+import unzip from "unzip-stream";
+
+config({
+  path: `${import.meta.dirname}/../.env`,
+});
+
+// 設定
+
+// 収集対象のリポジトリ
+const guestRepo = "voicevox/voicevox";
+// デプロイ情報を書き込むコメントの最初に付けるマーカー
+const commentMarker = "<!-- voiccevox preview info -->";
+// ダウンロードしたファイルを展開するディレクトリ
+const destinationDir = `${import.meta.dirname}/../public/preview`;
+// ビルドチェックの名前
+const pagesBuildCheckName = "update_preview";
+// ダウンロードするアーティファクトの名前
+const artifactName = "preview-page";
```

Comment:
（細かいけど）

リポジトリ名と違ってるとなんかミスりそうなので。。。
```suggestion
const artifactName = "preview-pages";
```

エディタの方に出してるプルリクエストの方にもサジェスト送ります！